lib/challenge.js

```diff
@@ -135,19 +162,38 @@ function prompt7() {
  */
 
 //  define someNumber here
-var someNumber;
+
+function getRandomArbitrary(min, max) {
+  let randomNumber = Math.random();
+  if (randomNumber === 0) {
+    randomNumber += 1;
+    return Math.floor(randomNumber * (max - min) + min);
+  } else {
+    return Math.floor(randomNumber * (max - min) + min);
+  }
+}
+let someNumber = getRandomArbitrary(1, 99);
 
```

Comment:
I appreciate the addition of the min and the max.  Nicely done reading the docs and expanding on the homework and using parameters for the function

Were you worried about a zero coming through still?

> `if (randomNumber === 0)`